assets/css/project/info.less

```diff
@@ -25,7 +31,6 @@
 		}
     }
 
-body p.source_info, /* cascade for weight */
```

Comment:
It seems to be some merging artifact, I'm not sure why it's here. Shell I remove it?